AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt

```diff
@@ -16,6 +16,7 @@ turkishairlines.com,atmos-tokyo.com,radarbox.com,ciceksepeti.com,rabota.by,temp-
 ! notifications - part of the functionality
 ! discord.com##div[class^="container"] > nav[class^="wrapper"] + div[class^="base"] > div[class^="notice-"]
 !
+trip.com##[class^=tcp-h5-install]
```

Comment:
It's better to add more specific rules.
1. Exchange your rule to these rules
```adblock
trip.com##.h5-install-banner
trip.com##.h5-home-install-bottom
```
2. Add this `trip.com##.tcp-h5-install-pop__reward` to the popup file.

Reply:
Note, that the rules from p.1 you should add under the comment.
```
! SECTION: Mobile apps - Mobile sites only
```

Reply:
Fixed it directly.